resource-watch/resource-watch
# Explore detail: Overview buttons

![imagen](https://user-images.githubusercontent.com/545342/75664163-4d3b9900-5c72-11ea-9dfd-96ca3cf2d0fd.png)
 
## Overview
This PR adds the dataset buttons from the overview section of Explore detail.

## Testing instructions
1. Check several datasets to verify that the buttons that appear correspond with the links/subscribable object available for each specific dataset. 
2. Verify that the subscription workflow is working in all scenarios. For that, you can use for instance the Fires dataset: http://localhost:9000/data/explore/for006nrt-Active-Fires-VIIRS_old_replacement 
The following cases should be tested:
- Create a subscription for an existing user area that has no subscriptions
- Create a subscription for an existing user area that already has some subscriptions created for it
- Create a subscription for a country the user has no area for
- Create a subscription selecting a country the user already has an area for
